core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1301,7 +1301,7 @@ private void waitForTableStateTransition(TableId tableId, TableState expectedSta
           lastRow = tablet.getExtent().toMetaRow();
 
           if (loc != null) {
-            serverCounts.increment(loc.getId(), 1);
+            serverCounts.increment(loc.toString(), 1);
```

Comment:
This seems like a regression. In general, unless you're printing something, it's better to use a more specific type than a String type, and a more specific method name than `toString()`, since `toString()` should not be relied upon to not change over time.

Reply:
They should be returning the same thing, but I don't think either method name is appropriate.  Looking at the impl, they both ultimately return `hostPort() + "[" + session + "]"`.  

Maybe something very explicit like `getHostPortSession()` would be better.  Then it will be clear what parts of the code we need to change if we drop/change the session.

Reply:
> Maybe something very explicit like `getHostPortSession()` would be better. Then it will be clear what parts of the code we need to change if we drop/change the session.

:+1: